content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+you can use Double Ducks to climb small humps without wasting time overshooting with a jump, but in addition to this they have 1 big advantage over jumps:
+**ducking is not considered a jump so the jump-land penalty does not proc.**  
+This means you can gain speed with air strafing without worrying about slowing down upon landing. This is the used in Count Jumps.   
+
+**Count Jump [1.6 Climb only]**: Done by first gaining speed with a prestrafe than double ducking to gain more speed in the air, land then finally jump. It can also be used in a Bhop sequence where doing another jump would result in over/under shooting the next platform, in this case the double duck is used as a "mini-jump" in the sequence and most be timed just like a Bhop with the first tap of duck being right as you land and the second tap being right after. This is also sometimes called a **Count Jump**. 
+
+**Duck Roll [1.6 Climb only]**: Sometimes called Silent Walking, Russian Walking, or G-strafe in Counter-Strike, Duck Roll is its given name in Half-life so for consistency's sake It will be called Duck Roll.  
+It is possible to do a "Bhop" sequence comprised entirely of Double Ducks. This lets you get pass the Bhop speed cap since it is not considered a jump. It is very hard to maintain Duck Rolls given how much more frequent you must duck compared to jumps in a Bhop sequence. Duck Rolls are vertically limited, so even with more speed they may not reach a place a Bhop can. DuckRoll's effectivness heavily depends on the map.  
 
-Climb works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**.
+**High Jump [1.6 Climb only]**: In Source and Goldsrc there exists a property called **edge friction** it decides what factor that "normal" friction will be multiplied by when a player is close to an edge. If it is set higher than 1, you will slow down every time you try to jump from the edge of a platform. These are the edge frictions for each Climb mode:  **TBD (MomClimb)**,     **1 (KZT)**,       **2 (1.6 Climb)**.  
+**[Notice that 1.6 Climb is the only Climb mode whose edge friction is diffrent than the normal friction]**.  
+You are considered to be next to an edge if both of these conditions are true:   
+**1. In the direction you are moving (not facing), an "invisible line" 16 units in front of the bottom-center of your collision box (your feet) does not intersect with anything solid**.  
```

Comment:
it's not a line actually, it checks the player's whole collision box 16 units in front of you and 70 units down, like you wrote in the following bits. it just checks exactly if that position is empty or not

the size of the checked box is the same as the player currently has; either ducked or standing.

very crudely edited your image to show what's really happening:
![image](https://github.com/user-attachments/assets/9177c2e4-c882-4fd1-a9f6-706b4d117161)